wiz-outpost-lite/templates/deployment.yaml

```diff
@@ -76,53 +81,54 @@ spec:
               secretKeyRef:
                 name: {{ .Values.agent.secretName }}
                 key: clientSecret
+          - name: OUTPOST_LITE_RUNNER_REGION
+            value: "{{ .Values.agent.outpostRegion }}"
+          {{- if .Values.service.enabled }}
+          - name: OUTPOST_LITE_RUNNER_METRICS_PORT
+            value: "{{ .Values.service.metricsPort }}"
+          {{- end }}
+          {{- if .Values.autoUpdate }}
+          - name: OUTPOST_LITE_RUNNER_AUTO_UPDATE
+            value: "1"
+          {{- end }}
           - name: http_proxy
             valueFrom:
-              secretKeyRef: 
+              secretKeyRef:
                 name: {{ .Values.httpProxyConfiguration.name | quote }}
                 key: "httpProxy"
                 optional: true
           - name: HTTP_PROXY
             valueFrom:
-              secretKeyRef: 
+              secretKeyRef:
                 name: {{ .Values.httpProxyConfiguration.name | quote }}
                 key: "httpProxy"
                 optional: true
           - name: https_proxy
             valueFrom:
-              secretKeyRef: 
+              secretKeyRef:
                 name: {{ .Values.httpProxyConfiguration.name | quote }}
                 key: "httpsProxy"
                 optional: true
           - name: HTTPS_PROXY
             valueFrom:
-              secretKeyRef: 
+              secretKeyRef:
                 name: {{ .Values.httpProxyConfiguration.name | quote }}
                 key: "httpsProxy"
                 optional: true
           - name: no_proxy
             valueFrom:
-              secretKeyRef: 
+              secretKeyRef:
                 name: {{ .Values.httpProxyConfiguration.name | quote }}
                 key: "noProxyAddress"
                 optional: true
           - name: NO_PROXY
             valueFrom:
-              secretKeyRef: 
+              secretKeyRef:
                 name: {{ .Values.httpProxyConfiguration.name | quote }}
                 key: "noProxyAddress"
                 optional: true
           - name: SSL_CERT_DIR
             value: "/usr/local/share/ca-certificates/:/certificates/"
-          args: [
-            "--region", "{{ .Values.agent.outpostRegion }}",
-            {{- if .Values.service.enabled }}
-            "--metricsPort", "{{ .Values.service.metricsPort }}",
-            {{- end }}
-            {{- if .Values.autoUpdate }}
-            "--autoUpdate",
-            {{- end }}
-          ]
```

Comment:
Move to env vars for future compat with runners that choose not to accept these args